script/cibuild

```diff
@@ -2,30 +2,9 @@
 # Usage: script/cibuild [--no-package]
 set -e
 
-# GHE appliance versions to run tests against. Remote metadata files are put in
-# place with this version at the beginning of each test and many commands have
-# conditional logic based on the remote version. Running the suite against
-# different major versions ensures we're covering these conditional paths.
-REMOTE_VERSIONS="2.19.0 2.21.0"
-
 # Enable verbose logging of ssh commands
 export GHE_VERBOSE_SSH=true
 
-# Run over all remote versions and run entire test suite against each
-res=true
-for version in $REMOTE_VERSIONS
-do
-  echo "==> Running testsuite with GHE_TEST_REMOTE_VERSION=$version"
-  export GHE_TEST_REMOTE_VERSION="$version"
-  if ! find test -name "test-*.sh" -print0 | xargs -0 -P 4 -n 1 /bin/bash; then
-    res=false
-  fi
-  echo
-done
-
-# If any of the version tests failed, exit non-zero
-$res
-
 # Bail out when --no-package given
 [ "$1" = "--no-package" ] && exit 0
```

Comment:
It looks like this `--no-package` flag was there to allow running the tests against different versions? Since those are now removed, perhaps this flag can be removed as well?

Reply:
Which would mean this can be removed, as well? https://github.com/github/backup-utils/blob/add-middle-version/Makefile#L3-L4

Is that right, @lildude ? I am still piecing together how this was used, so appreciate the extra set of eyes

Reply:
Looks like that's called by CI: https://github.com/github/backup-utils/blob/add-middle-version/.travis.yml#L14

Reply:
So I think we want to run the tests here, and that means leaving the `--no-package` flag

Reply:
That said, looking at the old Travis config, it looks like that flag is purely to prevent attempting to build the tarball and Debian pkg when building with `debuild` as it runs all tests before attempting to build the deb file and we don't want to include these files in the tarball or deb.

So the only time this section would be run is if manually without the flag. I'd say, keep this for now.

Reply:
Makes sense to me, thanks!